### PR TITLE
Top bar

### PR DESCRIPTION
**Pull request by andyOlarte514**
#Summary
- Add TopBar
- Add Profile (Initial Setup)
- Modify Styles

# Trello Card
- https://trello.com/c/VBIXij52

#Screenshots
image
image

tic tac toe

### CODE REVIEW

File: training-react/tic-tac-toe/package.json [hunk 1]
```diff
@@ -3,7 +3,12 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.6",
+    "@fortawesome/free-solid-svg-icons": "^5.4.1",
+    "@fortawesome/react-fontawesome": "^0.1.3",
     "apisauce": "^0.16.0",
+    "i": "^0.3.6",
```
Comment (henryzarza) [inline training-react/tic-tac-toe/package.json]: Delete this isn't necessary

File: training-react/tic-tac-toe/package.json [hunk 2]
```diff
@@ -3,7 +3,12 @@
...
+    "npm": "^6.4.1",
```
Comment (henryzarza) [inline training-react/tic-tac-toe/package.json]: The same here

File: training-react/tic-tac-toe/src/app/screens/Home/components/Header/index.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+import ROUTES from '@consts/route';
+import ActionCreators from '@redux/login/actions';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSignOutAlt } from '@fortawesome/free-solid-svg-icons';
+
+import styles from './styles.scss';
+
+function Header({ signOutUser }) {
+  return (
+    <header className={styles.header}>
```
Comment (henryzarza) [inline training-react/tic-tac-toe/src/app/screens/Home/components/Header/index.js]: Change this tag for `nav`

File: training-react/tic-tac-toe/src/app/screens/Home/components/Header/index.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+      <div className={styles.menuHeader}>
+        <div className={styles.logo}>
+          <Link to={ROUTES.GAME}>
+            <img className={styles.imgLogo} src={logo} alt="Wolox Logo" />
+          </Link>
+        </div>
+        <nav className={styles.navBar}>
```
Comment (henryzarza) [inline training-react/tic-tac-toe/src/app/screens/Home/components/Header/index.js]: Change this tag for a `div` and the same recommendation above

File: training-react/tic-tac-toe/src/app/screens/Home/components/Header/index.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSignOutAlt } from '@fortawesome/free-solid-svg-icons';
+
+import styles from './styles.scss';
+
+function Header({ signOutUser }) {
+  return (
+    <header className={styles.header}>
+      <div className={styles.menuHeader}>
+        <div className={styles.logo}>
```
Comment (henryzarza) [inline training-react/tic-tac-toe/src/app/screens/Home/components/Header/index.js]: Is it necessary this? I think that you could simplify the structure

File: training-react/tic-tac-toe/src/app/screens/Home/components/Header/styles.scss [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+  position: fixed;
+  top: 0;
+  width: 100%;
+
+  .menu-header {
+    align-items: center;
+    display: flex;
+    height: 100%;
+    justify-content: space-between;
+    width: 90%;
```
Comment (henryzarza) [inline training-react/tic-tac-toe/src/app/screens/Home/components/Header/styles.scss]: Why you do this? I think you can replace this for `padding` property

File: training-react/tic-tac-toe/src/app/screens/Home/components/Header/styles.scss [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+    .logo {
+      max-width: 250px;
+      width: 30%;
+
+      .img-logo {
+        height: auto;
+        width: 100%;;
+      }
+    }
+    .nav-bar {
```
Comment (henryzarza) [inline training-react/tic-tac-toe/src/app/screens/Home/components/Header/styles.scss]: Add a break line

File: training-react/tic-tac-toe/src/app/screens/Home/components/Header/styles.scss [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+      align-items: center;
+      display: flex;
+      flex-grow: 1;
+      flex-wrap: wrap;
+      flex-wrap: wrap;
```
Comment (henryzarza) [inline training-react/tic-tac-toe/src/app/screens/Home/components/Header/styles.scss]: This is repeated delete it

File: training-react/tic-tac-toe/src/app/screens/Home/components/Header/styles.scss [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+        border-bottom: 5px solid $blue-indigo;
+        color: $blue-indigo;
+        display: flex;
+        font-size: 17px;
+        font-weight: bold;
+        justify-content: center;
+        margin: 0 8px;
+        max-width: 100px;
+        min-height: 100%;
+        min-width: 80px;
```
Comment (henryzarza) [inline training-react/tic-tac-toe/src/app/screens/Home/components/Header/styles.scss]: You should assign only a of the 2 properties `max-width` or `min-width`

File: training-react/tic-tac-toe/src/app/screens/Home/components/Header/styles.scss [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+
+        &:hover {
+          border-color: $apple;
+          color: $apple;
+        }
+      }
+    }
+  }
+  .options-header {
```
Comment (henryzarza) [inline training-react/tic-tac-toe/src/app/screens/Home/components/Header/styles.scss]: Add a break line

File: training-react/tic-tac-toe/src/app/screens/Home/screens/Profile/index.js [hunk 11]
```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+
+function Profile() {
+  return <h1>Profile</h1>;
+}
```
Comment (henryzarza) [inline training-react/tic-tac-toe/src/app/screens/Home/screens/Profile/index.js:5]: Add a break line

File: training-react/tic-tac-toe/src/app/screens/Login/styles.scss [hunk 12]
```diff
@@ -31,7 +38,7 @@
     &:hover,
     &:active,
     &:focus {
-      background: $purple-amethyst;
+      background: $apple;
```
Comment (henryzarza) [inline training-react/tic-tac-toe/src/app/screens/Login/styles.scss]: Change this for `background-color` property

File: training-react/tic-tac-toe/src/scss/application.scss [hunk 13]
```diff
@@ -2,8 +2,13 @@
 @import './variables/colors';
 @import url(https://fonts.googleapis.com/css?family=Roboto:300);
 
+html,
```
Comment (henryzarza) [inline training-react/tic-tac-toe/src/scss/application.scss:5]: Delete this selector

File: training-react/tic-tac-toe/src/scss/application.scss [hunk 14]
```diff
@@ -2,8 +2,13 @@
...
+body,
+#root {
```
Comment (henryzarza) [inline training-react/tic-tac-toe/src/scss/application.scss:7]: Isn't a good approach to assign styles with id selectors

**Review by henryzarza — CHANGES_REQUESTED** [review]
You always forget the screenshots 😞

File: training-react/tic-tac-toe/src/app/screens/Home/components/Header/index.js [hunk 15]
```diff
@@ -0,0 +1,48 @@
...
+};
+
+const mapDispatchToProps = dispatch => ({
+  signOutUser: () => dispatch(ActionCreators.signOutUser())
+});
+
+export default connect(
+  null,
+  mapDispatchToProps
+)(Header);
```
Comment (wfolini) [inline training-react/tic-tac-toe/src/app/screens/Home/components/Header/index.js]: Split this component into a smart and dumb component

File: training-react/tic-tac-toe/src/app/screens/Home/components/Header/index.js [hunk 16]
```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { func } from 'prop-types';
```
Comment (wfolini) [inline training-react/tic-tac-toe/src/app/screens/Home/components/Header/index.js]: Don't apply destructuring in propTypes. Goes against to snippets

**Review by wfolini — CHANGES_REQUESTED** [review]
I need a GIF capture interaction between the screens routing for approve this PR. Please, upload it.

**Review by henryzarza — APPROVED** [review]

**Review by wfolini — APPROVED** [review]

**Pull request merged**